parsec/class/parsec_object.h

```diff
@@ -339,17 +340,26 @@ static inline parsec_object_t *parsec_obj_new_debug(parsec_class_t* type, const
 
 #define PARSEC_OBJ_CONSTRUCT(object, type)                             \
 do {                                                            \
-    PARSEC_OBJ_CONSTRUCT_INTERNAL((object), PARSEC_OBJ_CLASS(type));          \
+    PARSEC_OBJ_RELEASE_CONSTRUCT_INTERNAL((object), PARSEC_OBJ_CLASS(type), &parsec_obj_destruct); \
```

Comment:
`PARSEC_OBJ_CONSTRUCT_INTERNAL_WRELEASE` (or WITH_RELEASE?) the opposite direction is confusing.

Reply:
Done.